### PR TITLE
:memo: Assignments --- Redo file structure/layout

### PR DESCRIPTION
**Pull request by jameshughes89**
### Related Issues or PRs
Another another chip away at https://github.com/jameshughes89/cs102/issues/46

### What
Update the layout for the assignments . Also update gradle for the zip tasks. 

### Testing
Checked the gradle tasks and built the site with no _relevant_ errors or warnings. 

### Additional Notes
The zips and the folders the zips are from are still labelled "AssignmentX". This will be updated _after_ changes to the other repo and the naming and whatnot is done and updates can be carried over to this repo. 

In other words, I know they are left with the old names, but they will be adjusted later.

### CODE REVIEW

File: src/site/data/Assignment1/.idea/modules.xml [hunk 1]
```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (twentylemon) [inline src/site/data/Assignment1/.idea/modules.xml:1]: Do you still need these? Stuff in `.idea` contains info about how to build and run your project (like tests).

Comment (jameshughes89) [reply src/site/data/Assignment1/.idea/modules.xml:1]: I gave them the whole projects as a starting point for each, so they can open an "existing project". So it has src, test, resources all labelled.

Comment (twentylemon) [reply src/site/data/Assignment1/.idea/modules.xml:1]: /shrug ?

**Review by twentylemon — APPROVED** [review]

File: src/site/data/Assignment1/.idea/modules.xml (hunk 1 above)
Comment (jameshughes89) [reply src/site/data/Assignment1/.idea/modules.xml:1]: For now, yeah, let's go with that.

Nevertheless, it will be further looked into when assignment 1 is redone.
